tools/topology/topology2/include/components/kpb.conf

```diff
@@ -0,0 +1,89 @@
+#
+# Common widget kpb
+#
+# A generic kpb widget. All attributes defined herein are namespaced
```

Comment:
Can you please remove common and generic from all the patches in this series. There are not common and definitely not generic

Reply:
OK